tests/tls_e2e/host/host.cpp

```diff
@@ -60,20 +64,22 @@ typedef enum test_config_type
 
 oe_enclave_t* g_server_enclave = NULL;
 oe_enclave_t* g_client_enclave = NULL;
+
 int g_server_thread_exit_code = 0;
 int g_client_thread_exit_code = 0;
-pthread_mutex_t server_mutex;
-pthread_cond_t server_cond;
 bool g_server_condition = false;
-pthread_t server_thread_id;
+
+std::mutex server_mutex;
```

Comment:
It is better to use a consistent naming convention `g_` as the variables above seem to be doing.